Watchman.Discord/Areas/Messaging/Controllers/SendController.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Devscord.DiscordFramework.Commons.Exceptions;
+using Devscord.DiscordFramework.Framework.Architecture.Controllers;
+using Devscord.DiscordFramework.Framework.Commands;
+using Devscord.DiscordFramework.Framework.Commands.Parsing.Models;
+using Devscord.DiscordFramework.Middlewares.Contexts;
+using Devscord.DiscordFramework.Services;
+using Devscord.DiscordFramework.Services.Factories;
+using Watchman.Cqrs;
+using Watchman.Discord.Areas.Help.BotCommands;
+using Watchman.Discord.Areas.Help.Services;
+using Watchman.Discord.Areas.Messaging.BotCommands;
+using Watchman.DomainModel.Help.Queries;
+
+namespace Watchman.Discord.Areas.Messaging.Controllers
+{
+    public class SendController : IController
+    {
+        private readonly MessagesServiceFactory _messagesServiceFactory;
+        
+        public SendController(MessagesServiceFactory messagesServiceFactory)
+        {
+            this._messagesServiceFactory = messagesServiceFactory;
+        }
+        public async Task Send(SendCommand sendCommand, Contexts contexts)
+        {
+            var channelToSendMessageTo = sendCommand.Channel;
+            var messageToSend = sendCommand.Message;
+            if (messageToSend==null)
+            {
+                throw new NotEnoughArgumentsException();
```

Comment:
to powinno być ogarniane na wcześniejszym etapie
jeśli dostajemy komende 
i parametry nie są oznaczone jako [Optional]
to zakładamy że wszystkie parametry będą uzupełnione 

na razie to nie działa idealnie, ale jedna osoba już powoli się tym zajmuje

Reply:
ok, znaczy się, usunąć rzucanie tego wyjątku, tak?

Reply:
tak, usunąć rzucanie wyjątku wraz z sprawdzaniem tego
zakładamy że w tym momencie zawsze będą wszystkie argumenty podane

Reply:
ok, poprawione